magma/passes/ir.py

```diff
@@ -1,12 +0,0 @@
-from .passes import DefinitionPass
-from .clock import WireClockPass
-
-class IRPass(DefinitionPass):
-    def __init__(self, main):
-        super(IRPass, self).__init__(main)
-        self.code = ''
-
-        WireClockPass(main).run()
-
-    def __call__(self, definition):
-        self.code += repr(definition) + '\n\n'
```

Comment:
Is there a reason this was removed?  I think it could be useful to print out the entire representation of a circuit (including dependencies)